src/error-handling/try-operator.md

```diff
@@ -4,42 +4,47 @@ The try-operator `?` is used to return errors to the caller. It lets you turn
 the common
 
 ```rust,ignore
-match some_expression {
+let some_variable = match some_expression {
```

Comment:
This seems unnecessary for the point we're trying to make: that `foo?` expands to something else. It's not `let x = foo?` that expands, it's just the `foo?` part. As you know, you can use `?` in deeply nested expressions which don't match the pattern your showing here.

(Please take this change out)